pkg/certs/main.go

```diff
@@ -0,0 +1,47 @@
+package certs
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+func GetPKIRegistryInfo(platform, network string) (certgraphapi.PKIRegistryInfo, error) {
+	combined := certgraphapi.PKIRegistryInfo{}
+	chunkFileNames := exutil.FixturePaths("testdata", "tls", "ownership")
```

Comment:
avoid bindata.  Pass a path to this and read the files directly.

Reply:
Chunks grouped by owner are not stored there, but resulting TLS artifact registry is stored there (as its required to be readible by the test with no access to source)